Remotion/Web/Development.WebTesting/WebTestException.cs

```diff
@@ -25,6 +25,11 @@ public WebTestException (string message)
     {
     }
 
+    public WebTestException (string message, string browser, string browserVersion, string driverVersion)
+        : base ($"{message}\n(Browser: {browser}, version {browserVersion})\n(Webdriver version: {driverVersion})")

```

Comment:
Perhaps we should use `Environment.Newline` instead of `\n`, I think that's best practice

Reply:
actually, most of the time I just use `\r\n` because we're a) using Windows and thus always used Windows style and b) it's more readable than env.newline. Should we ever support Linux, we have to deal with this all over the place anyway. There are some projects with a more "universal" focus and a historical phase where we tried to use Env.Newline, but this just wasn't sustainable for the most part.